src/renderer/pages/project.vue

```
@@ -303,9 +290,10 @@
 import { useDebounceFn } from "@vueuse/core";
 import { useRoute, useRouter } from "vue-router";
 import { useI18n } from "vue-i18n";
+import { INITIAL_TITLE, INITIAL_DESCRIPTION } from "../../shared/constants";
```

ESLint

[error] 'INITIAL_TITLE' is defined but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)